lib/Request.php

```diff
@@ -275,18 +275,18 @@ function setRawServerData(array $data) {
      */
     function __toString() : string {
 
-        $out = $this->getMethod() . ' ' . $this->getUrl() . ' HTTP/' . $this->getHTTPVersion() . "\r\n";
-
+        $eol = PHP_EOL;
```

Comment:
Ok then I have to update just the unit test. Can you point me to the spec for future reference?